rspace/src/test/scala/coop/rchain/rspace/test/InMemoryStore.scala

```diff
@@ -181,6 +181,8 @@ class InMemoryStore[C, P, A, K] private (
         val wks  = _waitingContinuations.getOrElse(hash, Seq.empty[WaitingContinuation[P, K]])
         (cs, Row(data, wks))
     }.toMap
+
+  def getCheckpoint(): Blake2b256Hash = throw new Exception("unimplemented")
```

Comment:
`???` is shorthand for that

Reply:
I wanted to be a little more explicit about the fact that I didn't just accidentally forget to implement this.  But yes, you are right.

Reply:
I wasn't aware of that, sorry for being a smartass

Reply:
No need to apologize - I should be the one apologizing for my "But yes", which now reads a bit heavy-handed.

I would much rather you give comments like this than withhold them.  So thank you.